src/modules/controllers/spend.controller.js

```diff
@@ -0,0 +1,48 @@
+const Task = require('../../db/models/spend/index');
+
+module.exports.getAllSpends = (req, res, next) => {
+  Spend.find().then(result => {
+    res.send({data: result});
+  });
+};
+
+module.exports.createNewSpend =  (req, res, next) => {
+  const spend1 = new Spend(req.body);
+  const {shop, spend, date} = req.body;
```

Comment:
you first create an object, then you do destructurization, and only after that you check whether these fields exist.
isn't that strange?